test/e2e_source_map_test.ts

```diff
@@ -460,7 +1041,16 @@ function compile(
     return {compiledJS: '', dts: '', sourceMap: sourceMapTextToConsumer(''), sourceMapText: ''};
   }
 
-  const compiledJS = getFileWithName(options.outFile, jsFiles);
+  let outFile = options.outFile;
+  if (!outFile) {
```

Comment:
Unfortunately, tsc complains if you have imports or exports and specify an outFile.  That makes the sensible default undefined, but we still need the name of the outfile so we can get if from the compilation output, thus this code.